### DISCUSSION TITLE
New version: streams are detected, but downloadable files like mp4 not

### DISCUSSION THREAD

**Discussion by Fire222222** [aclap-dev/video-downloadhelper#222] [Help and Q&A] [answered]
### Your question

Well, before it was showing a sometimes very long list with all streams, mp4s etc., but now only the "best" three are listed and all of them are streams - the downloadable mp4s that were/are detected, are not showing anymore - not even if their quality is better than those of the best streams...

For now I have stopped automatic updates and installed video_downloadhelper-8.1.4.0.xpi.

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: accepted
key (only 16 first characters): 4fededac02214332
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\XXX",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by GrampaWildWilly** [comment]
You should post some URLs of pages on which you see those effects.  Also, you should click the link labeled Wiki at the top of this page & you should compare what you have with the latest releases of both the BETA version of VDH & the CoApp.  There's no sense in reporting a problem that might be solved in newer versions of the software.

**Reply by GrampaWildWilly** [reply]
Also, turning off automatic updates is the smart thing to do no matter what version of VDH you are on, as I have explained here #101 but also in these places: #52, #13, #1.  Just search for the word "automatic" within the text of those web pages.

**Reply by Fire222222** [reply]
Well, e. g. search for tubes. Many offer mp4s.

**Comment by paulrouget** [comment, marked as answer]
> the downloadable mp4s that were/are detected, are not showing anymore

In the settings, "More > Video Qualities", if you toggle "Ignore low quality videos", does it solve your problem?

**Reply by Fire222222** [reply]
You are right. It solves the issue. VDH believes that e. g. an 1 GByte big mp4 file is low quality and that not show it until you uncheck this "Ignore low quality videos". Strange as well is that the standard configuration is "Prefered container format: mp4", but VDH does not show mp4-files - at least not if there are streaming links detected.

**Reply by paulrouget** [reply]
Yeah, we assume that regular MP4 files are less interesting than streams. Which is 99% of the times true :)

I will add a button somewhere to show the hidden media.
